lib/rules/no-unused-expressions.js

```diff
@@ -95,12 +99,17 @@ module.exports = {
                     return isValidExpression(node.consequent) && isValidExpression(node.alternate);
                 }
             }
+
             if (allowShortCircuit) {
                 if (node.type === "LogicalExpression") {
                     return isValidExpression(node.right);
                 }
             }
 
+            if (allowTemplateTags) {
+                return (node.type === "TaggedTemplateExpression");
```

Comment:
If I'm reading this correctly, it seems enabling `allowTemplateTags: true` would cause all other types of statements to be disallowed. That's not intended, right?